### PR TITLE
Automatic coverage in run-unit-tests

### PR DESCRIPTION
**Pull request by ilaner**
## Related Issues
fixes:

### CODE REVIEW

File: demisto_sdk/commands/run_unit_tests/unit_tests_runner.py [hunk 1]
```diff
@@ -30,6 +33,31 @@
 POWERSHELL_RUNNER = f"{(Path(__file__).parent / 'pwsh_test_runner.sh')}"
 
 
+def coverage_report_editor(coverage_file: Path, code_file_absolute_dir: Path):
```
Comment (dorschw) [inline demisto_sdk/commands/run_unit_tests/unit_tests_runner.py:36]:
```suggestion
def fix_coverage_report_path(coverage_file: Path, code_path: Path):
```

File: demisto_sdk/commands/run_unit_tests/unit_tests_runner.py [hunk 2]
```diff
@@ -30,6 +33,32 @@
...
+        cursor = sql_connection.cursor()
+        files = cursor.execute("SELECT * FROM file").fetchall()
+        for id_, file in files:
+            file_name = Path(file).name
+            cursor.execute(
+                "UPDATE file SET path = ? WHERE id = ?",
+                (str(code_file_absolute_dir / file_name), id_),
+            )
+        sql_connection.commit()
+        logger.info("Done editing coverage report")
```
Comment (dorschw) [inline demisto_sdk/commands/run_unit_tests/unit_tests_runner.py:59]:
```suggestion
    logger.debug(f"Editing coverage report for {coverage_file}")
    with sqlite3.connect(coverage_file) as sql_connection:
        cursor = sql_connection.cursor()
        files = cursor.execute("SELECT * FROM file").fetchall()
        for id_, file in files:
            file_name = Path(file).name
            cursor.execute(
                "UPDATE file SET path = ? WHERE id = ?",
                (str(code_file_absolute_dir / file_name), id_),
            )
        sql_connection.commit()
        logger.debug("Done editing coverage report")
```

File: demisto_sdk/commands/run_unit_tests/unit_tests_runner.py [hunk 3]
```diff
@@ -30,6 +33,35 @@
...
+        cursor = sql_connection.cursor()
+        files = cursor.execute("SELECT * FROM file").fetchall()
+        for id_, file in files:
+            file_name = Path(file).name
+            cursor.execute(
+                "UPDATE file SET path = ? WHERE id = ?",
+                (str(integration_script_path / file_name), id_),
+            )
+        sql_connection.commit()
+        logger.debug("Done editing coverage report")
```
Comment (dorschw) [inline demisto_sdk/commands/run_unit_tests/unit_tests_runner.py:62]:
```suggestion
def fix_coverage_report_path(code_path: Path):
    """

    Args:
        code_path: Absolute path to the python file.

    Notes:
        the .coverage files contain all the files list with their absolute path.
        but our tests (pytest step) are running inside a docker container.
        so we have to change the path to the correct one.
    """
    coverage_file = code_path / ".coverage"
    if not coverage_file.exists():
        logger.debug(f"Skipping {integration_script_path} as it has no matching coverage report file (expected a file at {coverage_file}.")
        return
    logger.debug(f"Editing coverage report for {coverage_file}")
    with sqlite3.connect(coverage_file) as sql_connection:
        cursor = sql_connection.cursor()
        files = cursor.execute("SELECT * FROM file").fetchall()
        for id_, file in files:
            file_name = Path(file).name
            cursor.execute(
                "UPDATE file SET path = ? WHERE id = ?",
                (str(integration_script_path / file_name), id_),
            )
        sql_connection.commit()
        logger.debug("Done editing coverage report")
```

Comment (ilaner) [reply demisto_sdk/commands/run_unit_tests/unit_tests_runner.py]: Please take a look on my changes

**Review by dorschw — APPROVED** [review]

**Pull request merged**